src/m365/spo/commands/page/page-section-add.ts

```diff
@@ -46,7 +61,19 @@ class SpoPageSectionAddCommand extends SpoCommand {
       Object.assign(this.telemetryProperties, {
         order: typeof args.options.order !== 'undefined',
         zoneEmphasis: typeof args.options.zoneEmphasis !== 'undefined',
-        isLayoutReflowOnTop: !!args.options.isLayoutReflowOnTop
+        isLayoutReflowOnTop: !!args.options.isLayoutReflowOnTop,
+        isCollapsibleSection: typeof args.options.isCollapsibleSection !== 'undefined',
+        showDivider: typeof args.options.showDivider !== 'undefined',
+        iconAlignment: typeof args.options.iconAlignment !== 'undefined',
+        isExpanded: typeof args.options.isExpanded !== 'undefined',
+        gradientText: typeof args.options.gradientText !== 'undefined',
+        imageUrl: typeof args.options.imageUrl !== 'undefined',
+        imageHeight: typeof args.options.imageHeight !== 'undefined',
+        imageWidth: typeof args.options.imageWidth !== 'undefined',
+        fillMode: typeof args.options.fillMode !== 'undefined',
+        useLightText: typeof args.options.useLightText !== 'undefined',
+        overlayColor: typeof args.options.overlayColor !== 'undefined',
+        overlayOpacity: typeof args.options.overlayOpacity !== 'undefined'
       });
```

Comment:
We don't have to validate the type for booleans.


```suggestion
        isCollapsibleSection: !!args.options.isCollapsibleSection
        showDivider: !!args.options.showDivider,
        iconAlignment: typeof args.options.iconAlignment !== 'undefined',
        isExpanded: !!args.options.isExpanded,
        gradientText: typeof args.options.gradientText !== 'undefined',
        imageUrl: typeof args.options.imageUrl !== 'undefined',
        imageHeight: typeof args.options.imageHeight !== 'undefined',
        imageWidth: typeof args.options.imageWidth !== 'undefined',
        fillMode: typeof args.options.fillMode !== 'undefined',
        useLightText: !!args.options.useLightText,
        overlayColor: typeof args.options.overlayColor !== 'undefined',
        overlayOpacity: typeof args.options.overlayOpacity !== 'undefined'
      });
```

Reply:
Done

Reply:
@mkm17, looks like the suggestion wasn’t fully applied. No worries, I’ll take care of it during the merge. Just keep an eye on the suggestions next time!